run/test/specs/disappearing_video.spec.ts

```diff
@@ -1,86 +1,74 @@
 import { DISAPPEARING_TIMES } from '../../constants';
 import { androidIt, iosIt } from '../../types/sessionIt';
-import { DMTimeOption, DisappearActions } from '../../types/testing';
-import { sleepFor } from './utils';
+import { DMTimeOption } from '../../types/testing';
+import { runOnlyOnAndroid, runOnlyOnIOS, sleepFor } from './utils';
 import { newUser } from './utils/create_account';
 import { newContact } from './utils/create_contact';
 import { SupportedPlatformsType, closeApp, openAppTwoDevices } from './utils/open_app';
 import { setDisappearingMessage } from './utils/set_disappearing_messages';
 
-iosIt('Disappearing video message 1:1', disappearingVideoMessage1o1Ios);
-androidIt('Disappearing video message 1:1', disappearingVideoMessage1o1Android);
+iosIt('Disappearing video message 1:1', disappearingVideoMessage1o1);
+androidIt('Disappearing video message 1:1', disappearingVideoMessage1o1);
 
-async function disappearingVideoMessage1o1Ios(platform: SupportedPlatformsType) {
+const time: DMTimeOption = DISAPPEARING_TIMES.THIRTY_SECONDS;
+const timerType = 'Disappear after send option';
+const testMessage = 'Testing disappearing messages for videos';
+
+async function disappearingVideoMessage1o1(platform: SupportedPlatformsType) {
   const { device1, device2 } = await openAppTwoDevices(platform);
   // Create user A and user B
   const [userA, userB] = await Promise.all([
     newUser(device1, 'Alice', platform),
     newUser(device2, 'Bob', platform),
   ]);
-  const testMessage = 'Testing disappearing messages for videos';
   await newContact(platform, device1, userA, device2, userB);
-  await setDisappearingMessage(platform, device1, ['1:1', 'Disappear after read option'], device2);
-  // await device1.navigateBack(platform);
-  await device1.sendVideoiOS(testMessage);
+  await setDisappearingMessage(platform, device1, ['1:1', timerType, time], device2);
+  await runOnlyOnIOS(platform, () => device1.sendVideoiOS(testMessage));
+  await runOnlyOnAndroid(platform, () => device1.sendVideoAndroid());
   await device2.clickOnByAccessibilityID('Untrusted attachment message', 5000);
```

Comment:
you should really start using that stubbed `onIos().` syntax

Reply:
You mean instead of creating two functions sendVideoiOS and sendVideoAndroid?

Reply:
What needs to be done here @burtonemily?

Reply:
I'm not entirely sure what Audric meant here, I'm assuming he didn't like that I created two different functions instead of one function with the call `onIos` or `onAndroid` ? 
The only reason i split out the functionality is that there wasn't alot of shared functionality between the two platforms and so a combined function would just be two huge blocks with an if ios does this and if android do this... can combine them if needed. Just needed clarification

Reply:
Have created a ticket to remind me to discuss this once Audric is back 
https://optf.atlassian.net/browse/QA-354